src/tests/ftest/daos_test/daos_core_test.py

```diff
@@ -38,6 +38,6 @@ def test_subtest(self):
 
         Use Cases: core tests for daos_test
 
-        :avocado: tags=all,regression,vm,unittest,medium,daos_test
+        :avocado: tags=all,regression,vm,pr,hw,daos_test
```

Comment:
This seems to be reversing the very recent change of putting these tests on the VM cluster.  The most recent run is close.  1h26m vs 1h14m (respectively) but I am concerned about the IOR tests that are going to be added soon kicking that even more out of balance, towards even longer hardware runs.